README.md

```diff
@@ -500,8 +500,11 @@ can use most commands you know from the emacs package menu. This
 option requires the rust package `cargo-outdated` to be installed
 before being used.
 
-- `u` mark single crate for upgrade
-- `U` mark all upgradable crates
+- `u` mark single crate for upgrade. Asks using minibuffer the exact
+  version that should be used for upgrade.
+- `U` mark all upgradable crates.
+- `l` mark single crate for upgrading to latest available package
+- `L` mark all crates to latest available package
```

Comment:
I would also use version instead of available package here.

Reply:
Fixed, thanks!